digdag-standards/src/main/java/io/digdag/standards/operator/bq/BqDdlOperatorFactory.java

```diff
@@ -0,0 +1,285 @@
+            else {
+                TableConfig config;
+                try {
+                    config = objectMapper.readValue(node.traverse(), TableConfig.class);
+                }
+                catch (IOException e) {
+                    throw new ConfigException("Invalid table reference or configuration: " + node, e);
+                }
+                return table(defaultProjectId, defaultDataset, config);
+            }
+        }
+
+        private Table table(String defaultProjectId, Optional<DatasetReference> defaultDataset, TableConfig config)
+        {
+            Optional<String> datasetId = config.dataset().or(defaultDataset.transform(DatasetReference::getDatasetId));
+            if (!datasetId.isPresent()) {
+                throw new ConfigException("Bad table reference or configuration: Missing 'dataset'");
+            }
+            return new Table()
+                    .setTableReference(new TableReference()
+                            .setProjectId(config.project().or(defaultProjectId))
+                            .setDatasetId(datasetId.get())
+                            .setTableId(config.id()))
+                    .setSchema(config.schema().orNull())
+                    .setFriendlyName(config.friendly_name().orNull())
+                    .setExpirationTime(config.expiration_time()
+                            .transform(p -> p.getTimestamp().toInstant(request.getTimeZone()).toEpochMilli()).orNull())
+//                    .setTimePartitioning(config.time_partitioning().orNull())
+                    .setView(config.view().orNull());
+        }
+
+        @Override
+        public List<String> secretSelectors()
+        {
+            return ImmutableList.of("gcp.*");
+        }
+
+        @Override
+        protected TaskResult run(TaskExecutionContext ctx, BqClient bq, String projectId)
+
+        {
+            List<BqOperation> operations = Stream.of(
+                    params.getListOrEmpty("delete_datasets", JsonNode.class).stream().map(this::deleteDataset),
+                    params.getListOrEmpty("empty_datasets", JsonNode.class).stream().map(this::emptyDataset),
+                    params.getListOrEmpty("create_datasets", JsonNode.class).stream().map(this::createDataset),
+                    params.getListOrEmpty("delete_tables", JsonNode.class).stream().map(this::deleteTable),
+                    params.getListOrEmpty("empty_tables", JsonNode.class).stream().map(this::emptyTable),
+                    params.getListOrEmpty("create_tables", JsonNode.class).stream().map(this::createTable))
+                    .flatMap(s -> s)
+                    .collect(Collectors.toList());
+
+            int operation = state.params().get("operation", int.class, 0);
+            for (int i = operation; i < operations.size(); i++) {
+                state.params().set("operation", i);
+                BqOperation o = operations.get(i);
+                pollingRetryExecutor(state, "retry")
```

Comment:
[minor] Why don't we use other state key to avoid confusion with PollingRetryExecutor#RETRY?